autogpts/autogpt/autogpt/commands/web_search.py

```diff
@@ -134,7 +135,7 @@ def google(query: str, agent: Agent, num_results: int = 8) -> str | list[str]:
 
     except HttpError as e:
         # Handle errors in the API call
-        error_details = json.loads(e.content.decode())
+        error_details = json_loads(e.content.decode())
```

Comment:
do you have a reasonable suspicion that `e.content` might ever be dirty JSON?

Reply:
No, reverted